lib/MusicBrainz/Script/Utils.pm

```diff
@@ -17,7 +17,7 @@ Get a list of primary key column names for $schema.$table.
 =cut
 
 sub get_primary_keys($$$) {
-    my ($c, $schema, $table) = @_;
```

Comment:
How is the change to pass `sql` instead of `c` related to the fallback bit?

Reply:
In ProcessReplicationChanges we don't have access to a `$c`, just a `$sql` (luckily `get_primary_keys` doesn't require a `$c`).